src/util/expected.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+// clang-format off
+
```

Comment:
Do you prefer to keep the original formatting or disable formatting as I've done now? I guess we should try the real `std::expected` as well, then this isn't relevant anymore.

Reply:
Leave it like this for now.
We can discuss C++20 separately.